frontend/shared/src/hooks/useGoToFrontPage.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import useGoToPage from 'shared/hooks/useGoToPage';
+
+const useGoToFrontPage = (): (() => void) => {
+  const goToPage = useGoToPage();
+  return React.useCallback(() => goToPage('/'), [goToPage]);
```

Comment:
maybe adding to ROOTS enum is a good idea, cuz even root might change, like it happened to bf production site structure

Reply:
Perhaps this could be a good idea, but this one is in shared code as I thought that every service would have `/` as root.